ui/imports/shared/panels/EnterSeedPhrase.qml

```diff
@@ -34,15 +34,25 @@ ColumnLayout {
 
         property bool allEntriesValid: false
         property var mnemonicInput: []
+        property var incorrectWordAtIndex: []
         readonly property var tabs: [12, 18, 24]
         readonly property ListModel seedPhrases_en: BIP39_en {}
 
+        onIncorrectWordAtIndexChanged: {
+            if (d.incorrectWordAtIndex.length > 0) {
+                root.setWrongSeedPhraseMessage(qsTr("The phrase you’ve entered is invalid"))
+                return
```

Comment:
Before we return we have to call send `seedPhraseUpdated` signal, to notify the component which is using this one that the entered seed phrase is invalid. That covers the following scenario:
- user enters a valid seed phrase
- cause the seed phrase is valid we show some other component
- then the user changes a single word of the entered seed phrase and makes it invalid
- in case the signal is not sent, the component that is using it will still be displaying some other component as the seed phrase is still valid, because it doesn't know about the change

You can test this by trying to add a new account importing a new keypair.